src/styles/main.scss

```diff
@@ -1,3 +1,54 @@
+@import "src/styles/variables.scss";
+
+
 body {
   margin: 0;
 }
+
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  width: ($cell-size * $col-count) + $padding * 3;
+  gap: $gap;
+  padding: $padding;
+
+  &__day {
+    display: flex;
+    border: 1px solid black;
+    background-color: #eee;
+    width: $cell-size;
+    height: $cell-size;
+
+    @for $daysCount from 1 through 31 {
+      &:nth-child(#{$daysCount})::before {
+        font-family: Arial, Helvetica, sans-serif;
```

Comment:
Consider adding a font family for the entire page to get elements inherited from it